db/migrate/20240530180733_add_service_name_to_active_storage_blobs.active_storage.rb

```diff
@@ -0,0 +1,22 @@
+# This migration comes from active_storage (originally 20190112182829)
+class AddServiceNameToActiveStorageBlobs < ActiveRecord::Migration[6.0]
```

Comment:
You can remove this migration. AppDev students will not use active storage, or at the very least, they won't expect it to be installed/configured by default.